synspec_examples.ipynb

```diff
@@ -0,0 +1,722 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from synphot.models import Box1D\n",
+    "\n",
+    "wl_in_center = (wl_in_center * u.micron).to(u.angstrom)\n",
+    "band_width = spec_wavelengths[-1] - spec_wavelengths[0]\n",
+    "uniform_bp = SpectralElement(Box1D, amplitude=1, x_0=wl_in_center, width=band_width)\n",
```

Comment:
If you want a true uniform bandpass, `astropy.modeling.models.Const1D` is more appropriate.
```suggestion
    "uniform_bp = SpectralElement(Const1D, amplitude=1)\n",
```

Reply:
See https://github.com/tcjansen/GSoC/pull/17#issuecomment-511563088